presto-docs/src/main/sphinx/develop/serialized-page.rst

```diff
@@ -212,6 +212,20 @@ from the null flags.
 .. image:: serialized-page-row-column.png
   :width: 600
 
+DICTIONARY Encoding
```

Comment:
Should we add diagrams here and for RLE, similar to the other entries?

Reply:
It would nice, but I can't find the original PowerPoint I used to create these diagrams.

Reply:
No problem, but please add an issue and add the `docs` label post-merge so we don't lose track of it.